### ISSUE TITLE
MicroWin stops at license key

### ISSUE THREAD

**Issue by jandrews65** [ChrisTitusTech/winutil#3062]
## Describe the bug
When running the install with MicroWin, I have no license key to give since the license key is delivered when I connect to the domain. 

## To Reproduce
1. Reaching the screen where you enter the license key
2. Pressing "I have no license key" 
3. The install does not continue.
4. The install reverts to the boot screen again and loop around again.

## Expected behavior
I expected that when I click "I have no license key" the install will carry on as it does with the Microsoft original.

**Comment by SantiagoPintos** [comment]
Yes, I'm having the same problem, the installer aborts the installation with the following message "setup has failed to validate the product key".

**Comment by zr0fve** [comment]
![Image](https://github.com/user-attachments/assets/29f95449-0a41-413f-a5a3-324cd8f0ab21)

Same issue still as of Feb 2025. Not sure what is causing this.

**Comment by Real-MullaC** [comment]
@jandrews65 Do you know whether you can still get this issue?

**Comment by zany130** [comment]
https://github.com/ChrisTitusTech/winutil/issues/3422

apperently this can be bypassed with VirtualBox?

unfortuatly I cant test cuz I don't have access to virtual box. but the issue still happens in libvirt on linux on when running the iso on bare metal

**Comment by KevinStirling** [comment]
Also running into this issue on amd 5800x3d system, using an asus x570 tuf mobo
...
Following up to say I did not encounter this issue on a fresh stock win 11 install. I'm guessing it had to do with my lightly modified windows 10 install I was attempting to build the ISO on.

**Comment by genesis224** [comment]
![image.png](https://github.com/user-attachments/assets/8a24d069-6642-4457-847b-5374d741da4f)


in my end i just pressed this and it leads me to installing it without asking for keys. Alhough the ui is like a windows10/legacy version

**Comment by jeshua-gnnzlz** [comment]
i have the same problem

**Comment by JoaoMendesMartins** [comment]
Same problem here

**Comment by ghost** [comment]
fixed /close
